generators/server/generator.js

```diff
@@ -265,7 +265,6 @@ export default class extends ServerGenerator {
                 `import io.micronaut.core.annotation.Introspected;
 import java.io.Serializable;`,
               ),
-            content => content.replaceAll('jakarta.', 'javax.'),
```

Comment:
ah, there it is. Strange I couldn't find it when searching for `javax`. Thanks.

Reply:
I've tried to find too.